### PR TITLE
DO NOT MERGE adding dev journey blog post

### PR DESCRIPTION
**Pull request by joesepi**

### CODE REVIEW

File: dev-journey-blog.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+1. delete existing recommendations for this demo
+1. retrieve list of retailers
+1. identify retailers affected by the weather event
+1. retrieve their stock and make new shipments
+1. persist the recommendations
+
+And the last code block has an if/else that either rejects the promise **if** there is an error or **else** it resolves the promise with an object of recommendation information.
+
+If you look at the actions in the repo, you'll see the functional code is broken into logical separations of concern. Then, the recommendations interface is made available to our application via APIs that are native to the OpenWhisk framework on Bluemix. Separating the recommendations logic from the Acme Freight application cuts our costs, removes deployment dependencies and frees up our developers to focus on the core application needs.
```
Comment (ashumz) [inline dev-journey-blog.md]: link to part of repo with actions

File: dev-journey-blog.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+With Acme Freight, we chose to implement just the recommendations aspect using serverless, but there are companies that have moved their entire web architecture to serverless. The benefits are very easy to see and many organizations are embracing serverless. Serverless is also great for things beyond web applications, such as data processing, IoT, chatbots, and batch jobs.
+
+
+### Acme Freight + serverless
+
+Acme Freight uses a serverless approach to respond to weather events. An action is triggered by a storm event; retailers affected by the weather event are identified; their stock and shipments are retrieved; recommendations are made based on the storm, stock and shipments. While the guts of the code are abstracted out of this flow into their own function blocks, you can see the process in the following code from the recommend action:
```
Comment (ashumz) [inline dev-journey-blog.md]: add link to repo where this code comes from (recommend.js)

File: dev-journey-blog.md (hunk 1 above)
Comment (ashumz) [inline dev-journey-blog.md]: add call to action to learn more about deploying an openwhisk app in bluemix - make sure we are pitching the API side of this as much as the serverless side 

should we think about adding a section about securing the openwhisk api's to tie in the bluemix native side? @svennam92 @joesepi

**Review by ashumz — CHANGES_REQUESTED** [review]

File: dev-journey-blog.md (hunk 1 above)
Comment (joesepi) [reply dev-journey-blog.md]: I'll try something here.

**Review by ashumz — APPROVED** [review]

**Comment by joesepi** [comment]
Content approved. Work will continue in content platform. Closing this review.

**Pull request closed without merging**